test/update.test.js

```diff
@@ -0,0 +1,53 @@
+const gutil = require('gulp-util');
```

Comment:
Is this file testing the `update` method from the config.js file? If so I think it might be better to keep the tests here grouped inside config.test.js. :)